tests/jobs/localisation/example_case/scripts/init_test_case.py

```diff
@@ -0,0 +1,302 @@
+                + "/obs_"
+                + str(Iindx + 1)
+                + "_"
+                + str(Jindx + 1)
+                + "_"
+                + str(Kindx + 1)
+                + ".txt"
+            )
+
+            obs_file.write(f"GENERAL_OBSERVATION   OBS_{Iindx+1}_{Jindx+1}_{Kindx+1}  ")
+            obs_file.write("{ ")
+            obs_file.write(
+                f"DATA = RESULT_UPSCALED_FIELD ; INDEX_LIST = {obs_number} ; "
+            )
+            obs_file.write("RESTART = 0;  ")
+            obs_file.write(f"OBS_FILE = ./{obs_data_relative_file_name} ; ")
+            obs_file.write(" };\n")
+
+            data_file_name = observation_dir + "/" + obs_data_relative_file_name
+            print(f"Write file: {data_file_name} ")
+            with open(data_file_name, "w", encoding="utf8") as data_file:
+                data_file.write(f"{value}  {value_err}\n")
+
+
+def create_grid():
+    grid_file_name = settings.field.grid_file_name
+    nx, ny, nz = settings.field.grid_dimension
+    xsize = settings.grid_size.xsize
+    ysize = settings.grid_size.ysize
+    zsize = settings.grid_size.zsize
+    if settings.grid_size.use_eclipse_grid_index_origo:
+        flip = -1
+        x0 = 0.0
+        y0 = ysize
+        z0 = 0.0
+    else:
+        flip = 1
+        x0 = 0.0
+        y0 = 0.0
+        z0 = 0.0
+
+    dx = xsize / nx
+    dy = ysize / ny
+    dz = zsize / nz
+
+    grid_object = xtgeo.create_box_grid(
+        dimension=(nx, ny, nz),
+        origin=(x0, y0, z0),
+        increment=(dx, dy, dz),
+        rotation=0.0,
+        flip=flip,
+    )
+
+    polygon_file_name = settings.grid_size.polygon_file
+    if polygon_file_name is not None and os.path.exists(polygon_file_name):
+        print(f"Use polygon  file {polygon_file_name} to create actnum ")
+        polygon = xtgeo.polygons_from_file(polygon_file_name, fformat="xyz")
+        grid_object.inactivate_outside(polygon)
+
+    print(f"Write grid file: {grid_file_name} ")
+    grid_object.to_file(grid_file_name, fformat="egrid")
+    return grid_object
+
+
+def create_upscaled_grid():
+    grid_file_name = settings.response.grid_file_name
+    nx, ny, nz = settings.response.grid_dimension
+    xsize = settings.grid_size.xsize
+    ysize = settings.grid_size.ysize
+    zsize = settings.grid_size.zsize
+    if settings.grid_size.use_eclipse_grid_index_origo:
+        flip = -1
+        x0 = 0.0
+        y0 = ysize
+        z0 = 0.0
+    else:
+        flip = 1
+        x0 = 0.0
+        y0 = 0.0
+        z0 = 0.0
+
+    dx = xsize / nx
+    dy = ysize / ny
+    dz = zsize / nz
+
+    grid_object = xtgeo.create_box_grid(
+        dimension=(nx, ny, nz),
+        origin=(x0, y0, z0),
+        increment=(dx, dy, dz),
+        rotation=0.0,
+        flip=flip,
+    )
+
+    print(f"Write grid file: {grid_file_name} ")
+    grid_object.to_file(grid_file_name, fformat="egrid")
+    return grid_object
+
+
+def main(config_file_name=None):
```

Comment:
Seems like a `yaml` config-file is now required in order to run `init_test_case.py`, which mean we can specify the input parameter to `main` as non-optional, i.e., remove the `=None`.
Also, the `README.md` could perhaps also reflect the fact that `init_test_case.py` needs a `yaml` config-file.

Reply:
Done